### PR TITLE
add noo_dev_ocp.md

### PR DESCRIPTION
**Pull request by eranra**

### CODE REVIEW

File: noo_dev_ocp.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+- use `kubectl get pods` to observe the list of pods. Expect to see output such as
+```bash
+$ kubectl get pods
+NAME                                            READY   STATUS    RESTARTS   AGE
+goflow-kube-577d8c89b5-sll57                    1/1     Running   0          77s
+loki                                            1/1     Running   0          92m
+network-observability-plugin-676b6f8c8f-f62qz   1/1     Running   0          77s
+```
+
+- use `kubectl exec -it goflow-kube-757dcf5c74-8b24w bash` to start shell inside the `goflow-kube` pod
```
Comment (jpinsonneau) [inline noo_dev_ocp.md:52]: maybe specify to update the pod random id here ?

File: noo_dev_ocp.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+2. OCP cluster
+
+## Deployment process
+
+1. Connect to the OCP cluster using `oc login ...`
+2. execute `kubectl create namespace network-observability` - this creates the `network-observability` namespace
+3. execute `kubectl config set-context --current --namespace network-observability` - this changes the `kubectl` context to `network-observability` namespace
+4. Execute `./hack/deploy-loki.sh` - this  deploys a simple loki instance into the cluster
+4. Execute `./hack/deploy-grafana.sh` - this deploys a simple grafana instance into the cluster (user: admin password: admin)
+5. Execute `oc patch console.operator.openshift.io cluster --type='json' -p '[{"op": "add", "path": "/spec/plugins", "value": ["network-observability-plugin"]}]'` - this enables the console plugin
```
Comment (jpinsonneau) [inline noo_dev_ocp.md:17]: It works but a bit strange to run before the plugin being deployed ?

I usually do that at the end. Maybe just add a note about that is enough

File: noo_dev_ocp.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+1. Local linux machine
+2. OCP cluster
+
+## Deployment process
+
+1. Connect to the OCP cluster using `oc login ...`
+2. execute `kubectl create namespace network-observability` - this creates the `network-observability` namespace
+3. execute `kubectl config set-context --current --namespace network-observability` - this changes the `kubectl` context to `network-observability` namespace
+4. Execute `./hack/deploy-loki.sh` - this  deploys a simple loki instance into the cluster
+4. Execute `./hack/deploy-grafana.sh` - this deploys a simple grafana instance into the cluster (user: admin password: admin)
```
Comment (jpinsonneau) [inline noo_dev_ocp.md:16]: Do you plan to add some shells in this repo ? 

Here we use https://github.com/netobserv/documents/blob/main/loki_simple.md#hands-on
```
oc apply -f examples/zero-click-loki/1-storage.yaml
oc apply -f examples/zero-click-loki/2-loki.yaml
```

File: noo_dev_ocp.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+5. Execute `oc patch console.operator.openshift.io cluster --type='json' -p '[{"op": "add", "path": "/spec/plugins", "value": ["network-observability-plugin"]}]'` - this enables the console plugin
+6. Execute `make install` - this installs the operator CRDs into the cluster
+7. Execute `make deploy-sample-cr` - this installs an example of *flows.netobserv.io* instance into the cluster
```
Comment (jpinsonneau) [inline noo_dev_ocp.md:19]: Has been renamed to `make create-sample` in https://github.com/netobserv/network-observability-operator/pull/51

**Review by jpinsonneau — CHANGES_REQUESTED** [review]
@eranra do you feel the operator doc https://github.com/netobserv/network-observability-operator#deploy-an-existing-image needs update ? We can merge both if you prefer

File: noo_dev_ocp.md (hunk 4 above)
Comment (jpinsonneau) [reply noo_dev_ocp.md:19]: Ok just saw there is another PR => https://github.com/netobserv/network-observability-operator/pull/61

File: noo_dev_ocp.md (hunk 2 above)
Comment (eranra) [reply noo_dev_ocp.md:17]: done

File: noo_dev_ocp.md (hunk 3 above)
Comment (eranra) [reply noo_dev_ocp.md]: I changed the #PR https://github.com/netobserv/network-observability-operator/pull/61 so now this is changed to using the commands `make deploy-loki` (which behind the scenes using those yaml files)

:-)

File: noo_dev_ocp.md (hunk 4 above)
Comment (eranra) [reply noo_dev_ocp.md:19]: Exactly ... this pr assumes that  https://github.com/netobserv/network-observability-operator/pull/61 is merged :-)

File: noo_dev_ocp.md (hunk 1 above)
Comment (eranra) [reply noo_dev_ocp.md:55]: Done (added a note)

**Comment by eranra** [comment]
@jpinsonneau 

> @eranra do you feel the operator doc https://github.com/netobserv/network-observability-operator#deploy-an-existing-image needs an update? We can merge both if you prefer

It probably should ... but let's move step by step ... I think that it will make sense to split in "https://github.com/netobserv/network-observability-operator" the customer-facing text from the development text ... and update the text ... but this is a bit bigger ... and I do not feel that I have all knowledge (yet) to do that. let's do that in two different PR's

**Comment by eranra** [comment]
I added a `hold` tag until we merge https://github.com/netobserv/network-observability-operator/pull/61

**Review by jpinsonneau — APPROVED** [review]
:+1: looks good @eranra thanks for the changes

**Pull request merged**